index.js

```diff
@@ -74,7 +83,8 @@ const pseudoLocalization = (() => {
     }
   ) => {
     opts.blacklistedNodeNames = options.blacklistedNodeNames;
-    pseudoLocalize(document.body, options);
+    opts.strategy = options.strategy;
```

Comment:
This would introduce a breaking change. It's no longer possible to call start without a strategy key specified. 

`pseudoLocalization.start()` would pass but
`pseudoLocalization.start({blacklistedNodes [...]})` would fail.

We can solve this by better handling defaults and do a deep default value for the object and keys.

```js
  const start = (
    {
      strategy = "accented",
      blacklistedNodeNames = opts.blacklistedNodeNames
    } = {}
  ) => {
    opts.blacklistedNodeNames = blacklistedNodeNames;
    opts.strategy = strategy;
    pseudoLocalize(document.body);
    observer.observe(document.body, observerConfig);
  };
```